react-client/src/Components/App.js

```diff
@@ -59,6 +64,13 @@ function App() {
         });
 
         var eltarget = e.target;
+
+        var randomCode = "uniqueID"+uuidv4();
+        if(e.target.id === null || e.target.id.trim() === "")
+        {
+          e.target.setAttribute("id" , randomCode);                  
```

Comment:
Set attribute on `data-id` instead of `id` and use `data-id` to query the DOM later on

Reply:
noted already implemented